MediaJel/mediajel-tracker
# [FOR REVIEW] Implement Cypress e2e test suite

# Description

Impelemented basic locally running e2e test suite with cypress... We will configure the CI/CD pipeline later with Devops via the `docker-compose.yml` file. For now, we can build cypress tests locally.


## How to run the tests

Note: Ensure `localhost:1234` is available. You may use the `cypress` app locally via running `yarn run cypress:open`

0. Run `yarn install` to install the additional `cypress` dev dependency
1. Run the dev environment via `yarn start`, 
2. Run the tests via `yarn test`
3. At this stage `cypress` will run the spec located at `cypress/e2e/spec.cy.ts`. This is the location of our test file. You may make modifications to this test file for running tests
